GrandCentralBoard/Widgets/WebsiteAnalytics/WebsiteAnalyticsWidget.swift

```diff
@@ -25,7 +25,32 @@ final class WebsiteAnalyticsWidget: Widget {
     }
 
     func update(source: UpdatingSource) {
-        
+        switch source {
+        case let source as GoogleAnalyticsSource:
+            fetchAnalyticsFromSource(source)
+        default:
+            assertionFailure("Expected `source` as instance of `GoogleAnalyticsSource`.")
+        }
+    }
+
+    private func fetchAnalyticsFromSource(source: GoogleAnalyticsSource) {
+        source.read { [weak self] result in
+            switch result {
+            case .Success(let reports):
+                self?.updateWithReports(reports)
+            case .Failure:
+                break
+            }
+        }
     }
 
+    private func updateWithReports(reports: [PageViewsRowReport]) {
+        let reports = reports[0...min(3, reports.count)]
```

Comment:
We are showing max 3 rows in a table view. The view is designed like that, so it shouldn't go to configuration yet. WDYT?